creator/ingest_runs/migrations/0001_initial.py

```diff
@@ -1,24 +1,95 @@
-# Generated by Django 2.2.13 on 2020-11-06 18:20
+# Generated by Django 2.2.13 on 2020-11-09 22:49
 
+from django.conf import settings
 from django.db import migrations, models
+import django.db.models.deletion
+import uuid
 
 
 class Migration(migrations.Migration):
 
     initial = True
 
     dependencies = [
+        ("files", "0022_add_genomic_workflow_output_type"),
+        ("jobs", "0001_move_jobs"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
     ]
 
     operations = [
         migrations.CreateModel(
-            name='IngestRun',
+            name="IngestRun",
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('created_at', models.DateTimeField(help_text='Time when the ingest_run was created')),
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                (
+                    "input_hash",
+                    models.UUIDField(
+                        blank=True,
+                        editable=False,
+                        help_text="Identifies an ingest run by its input parameters. Autopopulated on save with the MD5 hash of all of the ingest run input parameters.",
+                        null=True,
+                    ),
+                ),
+                (
+                    "name",
+                    models.TextField(
+                        blank=True,
+                        help_text="The name of the ingest run. An field that is auto populated by concatenating the IDs of the IngestRun's file versions",
```

Comment:
`help_text="The name of the ingest run. A field that is autopopulated by concatenating the IDs of the IngestRun's file versions",`